assets/root-files/Db/phpunit.xml.dist

```diff
@@ -0,0 +1,102 @@
+<phpunit bootstrap="./test/bootstrap.php" colors="true">
+    <testsuites>
+        <testsuite name="zend-db Test Suite">
+            <directory>./test/</directory>
+        </testsuite>
+    </testsuites>
+
+    <groups>
+        <exclude>
+            <group>disable</group>
+        </exclude>
+    </groups>
+
+    <filter>
+        <whitelist addUncoveredFilesFromWhitelist="true">
+            <directory suffix=".php">./src</directory>
+        </whitelist>
+    </filter>
+
+    <php>
+        <ini name="date.timezone" value="UTC"/>
+
+        <!-- OB_ENABLED should be enabled for some tests to check if all
+             functionality works as expected. Such tests include those for
+             Zend\Soap and Zend\Session, which require that headers not be sent
+             in order to work. -->
+        <const name="TESTS_ZEND_OB_ENABLED" value="0" />
+
+        <!-- Note: the following is a FULL list of ALL POSSIBLE constants
+             currently in use in ZF2 for ALL COMPONENTS; you should choose the
+             constants appropriate to the component you are migrating. -->
+
+        <!-- Zend\Db\Adapter\Pdo\Mysql and Zend\Db\Adapter\Mysqli
+
+             There are separate properties to enable tests for the PDO_MYSQL
+             adapter and the native Mysqli adapters, but the other properties
+             are shared between the two MySQL-related Zend\Db adapters. -->
+        <const name="TESTS_ZEND_DB_ADAPTER_PDO_MYSQL_ENABLED" value="0" />
```

Comment:
@Maks3w - you're wrong; this is based off of current master, which still uses constants.